sql/item_mods.sql

```diff
@@ -25378,6 +25393,10 @@ INSERT INTO `item_mods` VALUES (21364,30,4);
 INSERT INTO `item_mods` VALUES (21366,8,3);
 INSERT INTO `item_mods` VALUES (21366,12,5);
 INSERT INTO `item_mods` VALUES (21366,23,10);
+INSERT INTO `item_mods` VALUES (21367,11,3); -- Tengu-no-Hane AGI+3
+INSERT INTO `item_mods` VALUES (21367,289,3); -- "Subtle Blow"+3
+-- INSERT INTO `item_mods` VALUES (21367,Daytime: Accuracy,15); -- Daytime: Accuracy+15
```

Comment:
@TeoTwawki would this be latent too? the daytime? or item script?

Reply:
yeah and that one exists pretty sure

Reply:
yup!
LATENT_TIME_OF_DAY = 26, //PARAM: 0: DAYTIME 1: NIGHTTIME 2: DUSK-DAWN
can go in latent sql table

Reply:
okay removed the 2 lines